mandatory/3-function-output.js

```diff
@@ -1,14 +1,18 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
   return Math.random() * 10;
+  // The math function returns a number between 0-9.
```

Comment:
Good try . We usually write comments to a function before the function declaration and body. Comments are written before the function because its 1) is a norm 2) based on this norm, modern IDEs and code documentation tools could provide help on usage, detect error in your code.